lua/modules/completion/config.lua

```diff
@@ -22,10 +22,21 @@ end
 
 function config.lua_snip()
 	local ls = require('luasnip')
+  local types = require "luasnip.util.types"
```

Comment:
The code style is different from the existing one.(Maybe need CI lint?)

```suggestion
    local types = require "luasnip.util.types"
```

Reply:
need add a stylua config.